dev/VSIX/Shared/WizardImplementation.cs

```diff
@@ -13,161 +13,159 @@
+        _waitDialog = ServiceProvider.GlobalProvider.GetService(typeof(SVsThreadedWaitDialog)) as IVsThreadedWaitDialog2;
+        if (_componentModel != null)
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
-            _componentModel = (IComponentModel)ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel));
-            _waitDialog = ServiceProvider.GlobalProvider.GetService(typeof(SVsThreadedWaitDialog)) as IVsThreadedWaitDialog2;
-            if (_componentModel != null)
-            {
-                _nugetProjectUpdateEvents = _componentModel.GetService<IVsNuGetProjectUpdateEvents>();
-                if (_nugetProjectUpdateEvents != null)
-                {
-                    _nugetProjectUpdateEvents.SolutionRestoreFinished += OnSolutionRestoreFinished;
-                }
-            }
-            // Assuming package list is passed via a custom parameter in the .vstemplate file
-            if (replacementsDictionary.TryGetValue("$NuGetPackages$", out string packages))
+            _nugetProjectUpdateEvents = _componentModel.GetService<IVsNuGetProjectUpdateEvents>();
+            if (_nugetProjectUpdateEvents != null)
             {
-                _nuGetPackages = packages.Split(';').Where(p => !string.IsNullOrEmpty(p));
+                _nugetProjectUpdateEvents.SolutionRestoreFinished += OnSolutionRestoreFinished;
             }
-        }        
-        public void ProjectFinishedGenerating(Project project)
+        }
+        // Assuming package list is passed via a custom parameter in the .vstemplate file
+        if (replacementsDictionary.TryGetValue("$NuGetPackages$", out string packages))
         {
-            ThreadHelper.ThrowIfNotOnUIThread();
-            _project = project;
-            Guid _projectGuid;
-            if (project != null)
-            {
-                Guid.TryParse(project.Kind, out _projectGuid);
+            _nuGetPackages = packages.Split(';').Where(p => !string.IsNullOrEmpty(p));
+        }
+    }        
+    public void ProjectFinishedGenerating(Project project)
+    {
+        ThreadHelper.ThrowIfNotOnUIThread();
+        _project = project;
+        Guid _projectGuid;
+        if (project != null)
+        {
+            Guid.TryParse(project.Kind, out _projectGuid);
 
-                if (_projectGuid.Equals(SolutionVCProjectGuid))
+            if (_projectGuid.Equals(SolutionVCProjectGuid))
+            {
+                ThreadHelper.JoinableTaskFactory.Run(async () =>
                 {
-                    ThreadHelper.JoinableTaskFactory.Run(async () =>
-                    {
-                        await InstallNuGetPackagesAsync();
-                    });
-                }
+                    await InstallNuGetPackagesAsync();
+                });
             }
         }
-        private async Task InstallNuGetPackagesAsync()
+    }
+    private async Task InstallNuGetPackagesAsync()
```

Comment:
```suggestion
    }
    
    private async Task InstallNuGetPackagesAsync()
```
nit: newline between functions

Reply:
newline added